### PR TITLE
Add /oasis-managers/json endpoint (SOFTWARE-3947)

### PR DESCRIPTION
**Pull request by edquist**
Hopefully this is everything we need!

Still need to test on topology-itb, but @matyasselmeci, hopefully you can let me know if anything looks obviously wrong  :D

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -222,6 +223,20 @@ def scitokens():
         return Response("Server error getting scitokens config, please contact help@opensciencegrid.org", status=503)
 
 
+@app.route("/oasis-managers/json")
+def oasis_managers():
+    if not _get_authorized():
+        return Response("Not authorized", status=400)
```
Comment (matyasselmeci) [inline src/app.py:229]:
```suggestion
        return Response("Not authorized", status=403)
```
Shouldn't it be 403 Forbidden?

Comment (edquist) [reply src/app.py:229]: Eh, sure!

**Comment by edquist** [comment]
Will remove do-not-merge label when this is working on topology-itb - there is still one more fix coming.

**Comment by edquist** [comment]
Had to fix up a couple bugs, but it works for me now on topology-itb:

https://topology-itb.opensciencegrid.org/oasis-managers/json?vo=MIS

->

`
[{"ContactID": "85fe79f7c8a447834460b6852f46d3eef32fb714", "Name": "Dave Dykstra", "DNs": ["/DC=com/DC=DigiCert-Grid/O=Open Science Grid/OU=People/CN=Dave Dykstra 508", "/DC=org/DC=cilogon/C=US/O=Fermi National Accelerator Laboratory/OU=People/CN=Dave Dykstra/CN=UID:dwd", "/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Dave Dykstra 508"], "CILogonID": "OSG1000005", "sshPublicKeys": ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDBBhR/0IPrMfKx4JWcsTeJUTilom0sohq3ezS525KRR1RD7JMUUVMv+/a9lYk3ixbcWlGan3TK+T91NZrRXunu7ddT23g2U/HrXmc07x8asPA/fKRPf3RTTNE5mn7+k0TJAKVcMML17ByyfxA2bYVA9WpHT4leiz5MWXgAKt4Y9+JzK9yM/QXGiQsF2chwxvRlXFml88b3WPMMslzFQX/YSTT/KQKKq0CHk2zgLHW+WXovormidCp0RFph5POli2ukd15KeMYaxJM93Eurz9/oNcRNYEzkks4kF3KLNwuyVPua+Uh8MhhbA8OMWwJsCbbt8QkkBbb47kImMgOYJLgR dwd@dwd.drdykstra.us"]}, {"ContactID": "ec1013224934d6a11a2a46a5234b3337095f5ec4", "Name": "Matyas Selmeci", "DNs": ["/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Matyas Selmeci", "/DC=org/DC=cilogon/C=US/O=University of Wisconsin-Madison/CN=Matyas Selmeci A148276"], "CILogonID": "OSG1000002", "sshPublicKeys": []}]
`

**Pull request merged**